tests/TestCase.php

```diff
@@ -39,12 +45,12 @@ protected function getEnvironmentSetUp($app)
     {
         $app['config']->set('websockets.apps', [
             [
-                'name' => 'Test App',
-                'id' => 1234,
-                'key' => 'TestKey',
-                'secret' => 'TestSecret',
+                'name'                   => 'Test App',
+                'id'                     => 1234,
+                'key'                    => 'TestKey',
+                'secret'                 => 'TestSecret',
                 'enable_client_messages' => false,
-                'enable_statistics' => true,
+                'enable_statistics'      => true,
```

Comment:
IMO the whitespace changes in this file aren't necessary, and are inconsistent with the rest of the codebase.

Reply:
You are 100% correct, this is my bad. My editor has a different code style set than this project. Easy fix.